README.md

```diff
@@ -5,7 +5,7 @@
 
 > ⚠️ This project is experimental and not suitable for production use. ⚠️
 
-Temporal CLI is a distribution of [Temporal](https://github.com/temporalio/temporal) that runs as a single process with zero runtime dependencies.
+Temporal CLI is a single-process binary that integrates the Temporal Server and UI suitable for development purposes. It also includes commands helpful for operating on Namespaces, Workflows, and other Temporal entities. 
```

Comment:
```suggestion
The Temporal CLI is a command-line interface for running Temporal Server and interacting with Workflows, Activities, Namespaces, and other parts of Temporal.
```

I think "single-process binary" is just relevant to people who are familiar with the old tools. I think it makes sense to craft our messaging for new users.